.test-defs/CreateShoot.yaml

```diff
@@ -20,6 +20,7 @@ spec:
     -cloud-profile-name=$CLOUDPROFILE
     -seed=$SEED
     -secret-binding=$SECRET_BINDING
+    -credentials-binding=$CREDENTIALS_BINDING
```

Comment:
Can we deprecate the `-secret-binding` and add a TODO to drop it in 3 releases. 
I don't see value in being able to specify both of them. The SecretBinding API will be soon or later deprecated, hence I would vote to remove the `-secret-binding` in 3 releases.
The only release to keep both of them atm is to ensure backwards compatibility IMO.

WDYT?

Reply:
🤔 I'd prefer to couple the deprecation of the `-secret-binding` flag to the deprecation of the API. As long as the API is supported officially, I would like to be able to run tests with it.